tests/functional_tests/ffront_tests/test_unrollPowOp.py

```diff
@@ -0,0 +1,139 @@
+import pathlib
+
+from eve.pattern_matching import ObjectPattern
+from functional.ffront import common_types as ct, field_operator_ast as foast
+from functional.ffront.func_to_foast import UnrollPowerOp
+
+
+def _make_power_testee(pow_n: str):
```

Comment:
```suggestion
def _make_power_testee(pow_n: int) -> foast.BinOp:
```
Let's make this an integer that is converted to a string below.